code/game/machinery/autolathe/matterforge.dm

```diff
@@ -0,0 +1,594 @@
+
+
```

Comment:
It would be nice if your code starts on line 1.

Reply:
ah shoot let me fix all this